File: go-sdk/pkg/models/operations/getanorganization.go
package operations

type GetAnOrganizationPathParams struct {
	Name string `pathParam:"style=simple,explode=false,name=name"`
}

type GetAnOrganizationRequest struct {
	PathParams GetAnOrganizationPathParams
}

type GetAnOrganizationResponse struct {
	ContentType                               string
	StatusCode                                int64
	GetAnOrganization200ApplicationJSONObject map[string]map[string]interface{}
}

File: python-sdk/src/planetscaleapi/models/operations/get_an_organization.py
from __future__ import annotations
import dataclasses
from typing import Any, Optional


@dataclasses.dataclass
class GetAnOrganizationPathParams:
    name: str = dataclasses.field(metadata={'path_param': { 'field_name': 'name', 'style': 'simple', 'explode': False }})
    

@dataclasses.dataclass
class GetAnOrganizationRequest:
    path_params: GetAnOrganizationPathParams = dataclasses.field()
    

@dataclasses.dataclass
class GetAnOrganizationResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    get_an_organization_200_application_json_object: Optional[dict[str, dict[str, Any]]] = dataclasses.field(default=None)
    